test/functional/mempool_packages.py

```diff
@@ -66,7 +66,10 @@ def run_test(self):
             assert_equal(mempool[x]['descendantcount'], descendant_count)
             descendant_fees += mempool[x]['fee']
             assert_equal(mempool[x]['modifiedfee'], mempool[x]['fee'])
+            assert_equal(mempool[x]['fees']['modified'], mempool[x]['fee'])
```

Comment:
fees.base = fee

Reply:
How did this passed in travis since `fees` is not in the response?

Reply:
Ah I think push/pr events because travis doesn't run `--extended` tests. Currently running this test gives:
```
$ test/functional/mempool_packages.py
...
2018-02-21 01:00:44.877000 TestFramework (ERROR): Key error
Traceback (most recent call last):
  File "/Users/promag/bitcoin/test/functional/test_framework/test_framework.py", line 125, in main
    self.run_test()
  File "test/functional/mempool_packages.py", line 69, in run_test
    assert_equal(mempool[x]['fees']['modified'], mempool[x]['fee'])
KeyError: 'fees'
```